setup.py

```diff
@@ -44,7 +44,14 @@
         "onnxruntime<1.15.0",
         "transformers>=4.34.0",
     ],
-    "openvino": ["openvino>=2023.2", "onnx", "onnxruntime", "transformers>=4.36.0", "optimum>=1.16.1"],
+    "openvino": [
+        "openvino>=2023.2",
```

Comment:
should OpenVINO be `>=2023.3`?

Reply:
Yes, but it will be resolved to `2023.3.0` because of `openvino-tokenizers` anyway.